db-models/comments.sql

```diff
@@ -0,0 +1,9 @@
+-- db-models/comments.sql
+
+CREATE TABLE comments (
+    comment_id SERIAL PRIMARY KEY,          -- Unique for each comment
+    parent_comment_id INTEGER,              -- ID of the parent comment for replies
+    author_name TEXT NOT NULL,              -- Author comment's name
+    content TEXT NOT NULL,                  -- Comment text
+    created_at TIMESTAMP DEFAULT NOW()      --Timestamp when comment was created
```

Comment:
The `DEFAULT NOW()` syntax actually doesn't exist in SQLite, we'll need to change it to `DEFAULT CURRENT_TIMESTAMP` https://stackoverflow.com/questions/200309/how-to-create-timestamp-column-with-default-value-now

Reply:
Okeyy!